checkov/secrets/runner.py

```diff
@@ -127,7 +129,17 @@ def run(
                     filter_ignored_paths(root, d_names, excluded_paths)
                     filter_ignored_paths(root, f_names, excluded_paths)
                     for file in f_names:
-                        if file not in PROHIBITED_FILES and f".{file.split('.')[-1]}" in SUPPORTED_FILE_EXTENSIONS:
+                        if runner_filter.secrets_scan_file_type:
+                            if 'all' in runner_filter.secrets_scan_file_type:
+                                if 'Dockerfile' in file or f".{file.split('.')[-1]}" in (ADDED_TO_SECRET_SCAN_FILES_TYPES + SUPPORTED_FILE_EXTENSIONS):
```

Comment:
just create a new constant for `(ADDED_TO_SECRET_SCAN_FILES_TYPES + SUPPORTED_FILE_EXTENSIONS)` at the top level, so it won't be recreated each time.